invenio_rdm_records/cli.py

```diff
@@ -39,6 +42,86 @@ def demo():
     click.secho('Created records!', fg='green')
 
 
+@rdm_records.command('demo_submission_requests')
+@click.option("-u", "--user", type=click.STRING)
+@with_appcontext
+def demo_submission_requests(user=system_identity.id):
+    """Create 100 fake submission requests for demo purposes."""
+    click.secho('Creating demo requests...', fg='green')
+
+    # Create 25 drafts in each community.
+    n_drafts = 25
+    # Create 4 communties.
+    for _ in range(4):
+        fake_community = create_fake_community()
+        fake_drafts = []
+        for _ in range(n_drafts):
+            fake_drafts.append(create_fake_draft())
+        create_demo_community_submission_request.delay(
+            user,
+            fake_community,
+            fake_drafts,
+            n_drafts=n_drafts,
+            n_requests=1
+        )
+
+    click.secho('Created requests!', fg='green')
+
+
+@rdm_records.command('demo_membership_requests')
+@click.option("-u", "--user", type=click.STRING)
+@with_appcontext
+def demo_membership_requests(user=system_identity.id):
+    """Create 50 fake membership requests for demo purposes."""
+    click.secho('Creating demo requests...', fg='green')
+
+    # Create 25 requests in each community.
+    n_requests = 25
+
+    # Create 2 communties.
+    for _ in range(2):
+        fake_community = create_fake_community()
+        fake_requests = []
+        for _ in range(n_requests):
+            fake_requests.append(create_fake_request())
+
+        create_demo_community_membership_request.delay(
+            user,
+            fake_community,
+            fake_requests,
+            n_requests=n_requests,
+        )
+
+    click.secho('Created requests!', fg='green')
+
+
+@rdm_records.command('demo_invitation_requests')
+@click.option("-u", "--user", type=click.STRING)
+@with_appcontext
+def demo_invitation_requests(user=system_identity.id):
```

Comment:
Same comment on this func here.

I would rather split all this in inner private func, but only have one `demo`. As a dev/user, I have no idea on which order I should run these various commands.